torch/csrc/lazy/ts_backend/ts_eager_fallback.cpp

```diff
@@ -187,9 +226,14 @@ void ts_eager_fallback(
       // TensorList args onto the CPU at the same time. We can improve this if
       // we need better perf for XLA's CPU fallbacks.
       auto eager_ivalue = c10::IValue(c10::List<at::Tensor>(
-          to_eager(ivalue.toTensorList().vec(), device_type)));
+          to_eager(ivalue.toTensorVector(), device_type)));
       (*stack)[arguments_begin + idx] = std::move(eager_ivalue);
       tensorlist_args.push_back(ivalue.toTensorList());
+    } else if (ivalue.isOptionalTensorList()) {
+      auto eager_ivalue = c10::IValue(c10::List<c10::optional<at::Tensor>>(
+          to_eager(ivalue.toOptionalTensorVector(), device_type)));
+      (*stack)[arguments_begin + idx] = std::move(eager_ivalue);
+      opt_tensorlist_args.push_back(ivalue.toOptionalTensorList());
```

Comment:
cc @wconstab, I think this should fix `index_put_` for LTC

Reply:
I forgot (if I knew) what the problem with index_put_ was for LTC.  Is this fixing something that is broken when index_put falls back?